metrics.go

```diff
@@ -33,40 +33,40 @@ type Metrics struct {
 	TracesJoinedNotSampled metrics.Counter `metric:"traces" tags:"state=joined,sampled=n"`
 
 	// Number of sampled spans started by this tracer
-	SpansStartedSampled metrics.Counter `metric:"spans_started" tags:"sampled=y"`
+	SpansStartedSampled metrics.Counter `metric:"started_spans" tags:"sampled=y"`
```

Comment:
Curious why this change? and similarly for `finished_spans`

Reply:
wanted to normalize all names to end with noun